src/IHT.jl

```diff
@@ -7,7 +7,6 @@ using DataFrames
 using Gadfly
 using MendelBase
 using SnpArrays
-using StatsBase
```

Comment:
wait I think that other functions in IHT.jl rely on this, maybe `cv_get_num_folds` or something like that?